analysis/paper/paper.Rmd

```diff
@@ -60,6 +61,7 @@ Here is a citation [@Marwick2017]
 # Acknowledgements
 
 # References 
+<div id="refs"></div>
```

Comment:
This paper.Rmd template is supposed to be used also for rendering to latex or word, ja? I don't think a specific html solution should be in the general template. Could be mentioned in the Readme, but not imposed on everyone. On the other hand the header is already prepared for html...

Reply:
You are right. I changed it to be commented out with a description how to use when needed.